test/testsuite/Spec/Common.hs

```diff
@@ -0,0 +1,48 @@
+module Spec.Common (agesAgo, testUser, testUserEntity) where
+
+import           Data.Time as Time
+import qualified Database.Persist as Database
+import qualified Database.Persist.Sql as Database
+
+-- ⚛️
+
+import           Fission.Models
+import qualified Fission.User.Role.Types as User.Roles
+import           Fission.Prelude
+
+
+-- TIME
+
+
+agesAgo :: UTCTime
+agesAgo = UTCTime
+  { utctDay = ModifiedJulianDay { toModifiedJulianDay = 0 }
+  , utctDayTime = Time.secondsToDiffTime 0
+  }
+
+
+
+-- USERS
+
+
+testUser :: User
+testUser = User
+  { userUsername = "testUser"
+  , userEmail = Just "test@fission.codes"
+  , userRole = User.Roles.Regular
+  , userActive = True
+
+  --
+  , userHerokuAddOnId = Nothing
+  , userSecretDigest = "SECRET_DIGEST"
+
+  --
+  , userInsertedAt = agesAgo
+  , userModifiedAt = agesAgo
+  }
+
+testUserEntity :: Database.Entity User
+testUserEntity = Database.Entity
+  { entityKey = Database.toSqlKey 0
+  , entityVal = testUser
+  }
```

Comment:
@icidasset yup these fixtures work for now 👌 

In general, fixtures are pretty brittle / only handle the one case. This can lead to sub-optimal tests that rely on these specific cases. This is equally true for Elixir, Ruby, JS, and so on. In those languages I generally recommend starting with a using a faker of some kind. Haskell has one: https://www.stackage.org/lts-14.20/package/fakedata-0.2.2.

However, since we're also going to be prop testing, we probably want to have them as instances of QuickCheck's `Arbitrary` typeclass. Here's an example:

```haskell
newtype Prime a = Prime a deriving Show

instance (Integral a, Arbitrary a) => Arbitrary (Prime a) where
  arbitrary = do
    x <- frequency [ (10, choose (0, 1000))
                   , (5, choose (1001, 10000))
                   , (1, choose (10001, 50000))
                   ]
    return $ Prime (primes !! x)

-- Random value outside of QuickCheck:

randomPrime :: Prime <- generate
```

Let's merge this for now, though.